### ISSUE TITLE
change/reset fido 2 pin

### ISSUE THREAD

**Issue by si458** [trustcrypto/OnlyKey-App#222]
it would be nice to have in the app the ability to change the FIDO 2 pin!
this is possible by the CLI according to the docs https://docs.onlykey.io/command-line.html#wipemode-num
but dont see it anywhere in the desktop app!
i accidently set a pin when testing on my android device with chrome,
then suddenly now my desktops are asking for the fido 2 pin!
yet when i first started using it, i was never asked to set a pin at all to use fido 2

**Comment by onlykey** [comment]
Yes, when you enable FIDO2 on device it will ask you to set a FIDO2 pin, then you will have to enter this pin in the future to use FIDO2. Changing this pin is a feature that is built-in to Windows. You can add or change FIDO2 pin by going here:
![image](https://github.com/user-attachments/assets/b4d60eca-061c-4bbb-a594-0d9161e2f165)
Then you will have the option to add or change pin:
![image](https://github.com/user-attachments/assets/b09a3cbe-9cc9-424a-9dd8-0da963ee6312)

**Comment by si458** [comment]
@onlykey im sadly on mac so i dont have this option
and when i setup my fido 2 using firefox, i wasnt prompted to enter a fido 2 pin and never needed one
it was only because i plugged it into my phone and used chrome to login to my test website
that it randomly asked me to create a pin, so i did as `12345678` as i had NO IDEA what the pin was for?
then realised afterwards what it was and couldnt change it!

**Comment by onlykey** [comment]
@si458 You can also change the FIDO2 PIN using the Chrome web browser:

![image](https://github.com/user-attachments/assets/a2ba1e0b-87e9-474d-9144-f688fa440ea7)

**Comment by onlykey** [comment]
I don't think there is anyway to do this with Firefox unfortunately

**Comment by gp1019** [comment]
I also have an issue with this.

I haven't seen a warning anywhere on the documentation stating that we have to configure yet _another_ PIN on the Onlykey to be able to login to websites and put that same PIN for each account that you want to add/login the Onlykey to. I don't think Linux even has a way to change the PIN on it either.

What's worse is that I attempt to login to a website that asked for a PIN on Android with the adapter I ordered. Turns out Android registers the Onlykey as an external keyboard, so I'm limited to 6 buttons. I can't change to an onscreen keyboard at all for me to put a PIN in. I've attempted to look into the settings and developer settings to see if there's a way to override it, but nothing came up (unless if there's some ADB command I'm not aware of). Only workaround I found is having a preconfigured PIN/Password stored in one of the slots to be able put in a PIN.
